classes/class-loader.php

```diff
@@ -356,7 +356,8 @@ public function enqueue_editor_scripts( $hook ) {
 
 		// Variables reflected from PHP to JS.
 		$reflection_vars = [
-			'dateFormat' => get_option( 'date_format' ),
+			'dateFormat'                => get_option( 'date_format' ),
+			'enable_analytical_cookies' => $option_values['enable_analytical_cookies'] ?? '',
```

Comment:
Shouldn't we also check this value in the editor, and display a notice that what is entered won't be shown on the front end until the option is enabled?